item.rb

```diff
@@ -1,20 +1,48 @@
 class Item
-  attr_reader :id
-  attr_accessor :genre, :author, :source, :label, :publish_date
+  attr_reader :archived, :genre
+  attr_accessor :author, :source, :label, :publish_date
 
-  def initialize(publish_date, archived: false)
-    @id = rand(1_000)
+  def initialize(publish_date)
```

Comment:
@deniodev 
Although I like the fact that you have removed all unnecessay initialization parameters, can you add `archived=false` into the parameters instead of automatically asigning it to default. In this case you will be giving the chance to mark an Item as archived or not.